packages/cloud-cognitive/src/components/Datagrid/Datagrid/DatagridVirtualBody.js

```diff
@@ -44,8 +46,23 @@ const DatagridVirtualBody = (datagridState) => {
     page,
     handleResize,
     withOverflowRow,
+    tableId
   } = datagridState;
 
+  const gridElement = document.querySelector(`#${tableId}`);
```

Comment:
I would convert this to `useRef` so it doesn't recreate this variable on every render

Reply:
Updated to use `useRef`